### PR TITLE
put deprecation message GET /user/ v1

### PR DESCRIPTION
**Pull request by remenska**

### CODE REVIEW

**Review by twessling-icas — APPROVED** [review]

File: doc/next/_static/doctools.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *
  * Sphinx JavaScript utilities for all documentation.
  *
- * :copyright: Copyright 2007-2017 by the Sphinx team, see AUTHORS.
+ * :copyright: Copyright 2007-2014 by the Sphinx team, see AUTHORS.
```
Comment (twessling-icas) [inline doc/next/_static/doctools.js:7]: wait whut

File: doc/next/_static/doctools.js [hunk 2]
```diff
@@ -91,30 +91,6 @@ jQuery.fn.highlightText = function(text, className) {
   });
 };
 
-/*
- * backward compatibility for jQuery.browser
- * This will be supported until firefox bug is fixed.
- */
-if (!jQuery.browser) {
```
Comment (twessling-icas) [inline doc/next/_static/doctools.js:98]: hmmm what's your version of sphinx

File: doc/next/_static/doctools.js (hunk 1 above)
Comment (remenska) [reply doc/next/_static/doctools.js:7]: oh wow :D

File: doc/next/_static/doctools.js (hunk 2 above)
Comment (remenska) [reply doc/next/_static/doctools.js:98]:
```
$ sphinx-build --version
Sphinx (sphinx-build) 1.5.5
```

Comment (remenska) [reply doc/next/_static/doctools.js:98]: I will merge anyhow, don;'t see that copyright 2014 exposed anywhere, and I seem to have the correct sphinx version.

**Pull request merged**